pkg/store/github/github.go

```diff
@@ -12,6 +12,7 @@ import (
 	"sync"
 
 	"github.com/google/go-github/v43/github"
+	"github.com/hashicorp/go-version"
```

Comment:
Jeg foreslår å gi denne et annet navn, slik at vi ikke har `version` liggende definert med en package inni denne pakken her. Hva synes du om `goversion "github.com/hashicorp/go-version"`?

Reply:
Sounds good, but don't know how to do that :)

Reply:
changed